certbot/docs/using.rst

```diff
@@ -548,6 +548,30 @@ revocation from any ACME account::
 
   certbot revoke --cert-path /etc/letsencrypt/live/example.com/cert.pem --key-path /etc/letsencrypt/live/example.com/privkey.pem
 
+
+.. _ecdsa_account_keys:
+
+ECDSA account keys
+~~~~~~~~~~~~~~~~~~
+
+As of Certbot 2.0, accounts can be created with a ``secp256r1`` curve.
```

Comment:
Not sure about the timeline to add this. It won't create breaking changes, so can go in 1.X or 2.X.